go.mod

```diff
@@ -115,3 +115,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
+
+replace sigs.k8s.io/karpenter => github.com/jigisha620/karpenter v0.0.0-20240403195503-a1d90e788c2a
```

Comment:
We should remove this line

Reply:
I am going to remove this once the upstream PR gets merged. This has been added because of the dependency on upstream changes for which I have a PR out - https://github.com/kubernetes-sigs/karpenter/pull/1164
The RetryableError added in upstream PR is being referenced in this one.

Reply:
Removed